.docker/Dockerfile-build

```diff
@@ -1,44 +1,28 @@
-FROM golang:1.19-alpine3.18 AS builder
+FROM golang:1.19 AS builder
 
-RUN apk -U --no-cache --upgrade --latest add build-base git gcc bash
+RUN apt-get update && apt-get upgrade -y &&\
+    mkdir -p /var/lib/sqlite
 
 WORKDIR /go/src/github.com/ory/hydra
-RUN mkdir -p ./internal/httpclient
-
-COPY go.mod go.sum ./
-COPY internal/httpclient/go.* ./internal/httpclient/
 
 ENV GO111MODULE on
 ENV CGO_ENABLED 1
 
-RUN go mod download
-
 COPY . .
+RUN go mod download &&\
+    go build -tags sqlite,json1 -o /usr/bin/hydra
 
-RUN go build -tags sqlite,json1 -o /usr/bin/hydra
```

Comment:
Please keep these instructions as they were - they prevent excessive downloading of go modules. Thanks!